backend/src/test/java/de/neuefische/backend/ChoreoServiceTest.java

```diff
@@ -0,0 +1,119 @@
+package de.neuefische.backend;
+
+import de.neuefische.backend.choreo.Choreo;
+import de.neuefische.backend.choreo.ChoreoDTO;
+import de.neuefische.backend.choreo.ChoreoRepo;
+import de.neuefische.backend.choreo.ChoreoService;
+import de.neuefische.backend.move.Move;
+import de.neuefische.backend.move.MoveService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+class ChoreoServiceTest {
+
+    private ChoreoRepo choreoRepo;
+    private MoveService moveService;
+    private ChoreoService choreoService;
+
+    @BeforeEach
+    public void setUp() {
+        choreoRepo = mock(ChoreoRepo.class);
+        moveService = mock(MoveService.class);
+        choreoService = new ChoreoService(choreoRepo, moveService);
+    }
+
+    @DirtiesContext
+    @Test
+    void getAllChoreos_ShouldReturnListOfAllChoreos() {
+        //GIVEN
+        Choreo choreo1 = new Choreo("1", "Choreo 1", List.of("1", "2", "3"));
+        Choreo choreo2 = new Choreo("2", "Choreo 2", List.of("1", "2", "3", "2", "1"));
+        List<Choreo> expectedChoreos = Arrays.asList(choreo1, choreo2);
+
+        when(choreoRepo.findAll()).thenReturn(expectedChoreos);
+        when(choreoRepo.findById("1")).thenReturn(Optional.of(choreo1));
+        when(choreoRepo.findById("2")).thenReturn(Optional.of(choreo2));
```

Comment:
du rufst die findById() jetzt nicht mehr in getAllChoreoDTOsByAllChoreos() auf, weswegen diese when/theReturns für findById nicht mehr gebraucht werden